pkg/lib/machine-change-auth.html

```diff
@@ -82,6 +79,17 @@ <h4 class="modal-title" translate="yes">Log in to {{host}}</h4>
                 </td>
             </tr>
             {{/show_locked_identity}}
+            <tr>
+              <td translate="yes">SSH Key</td>
+              <td>
+                <div class="checkbox">
+                  <label>
+                    <input type="checkbox" id="login-setup-keys"/>
+                    <span translate="yes">Automatically log in via SSH public key authentication in the future.</span>
```

Comment:
As mentioned above, might need some wording around the fact that it will add ssh public keys.

Reply:
Something like:
`"Add public SSH key to remote host. Will allow you to connect to it without password in the future."`

Reply:
Done.